var/spack/repos/builtin/packages/py-hiredis/package.py

```diff
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack import *
+
```

Comment:
@rvinaybharadwaj huh?

Reply:
@junghans The commits have been clobbered when I separated the pull requests. Probably resolving merge conflict here.